tests/routes/serialization/reimbursement_csv_test.py

```diff
@@ -116,7 +116,7 @@ def test_generate_payment_csv_raw_contains_human_readable_status_with_details_wh
         raw_csv = ReimbursementDetails(payments_info[0]).as_csv_row()
 
         # then
-        assert raw_csv[13] == "Erreur d'envoi du remboursement : Iban non fourni"
+        assert raw_csv[13] == "Remboursement en cours : Iban non fourni"
 
 
 def _get_header(csv, line):
```

Comment:
Ça existait avant ta PR, mais :

```suggestion
def _get_line(csv, line):
```

... puisque ça renvoie une ligne et pas le header :)

Reply:
Tout a fait.

Reply:
Done